mysql-test/suite/sys_vars/t/pxc_strict_mode_basic.test

```diff
@@ -0,0 +1,50 @@
+#
+# Test different setting option for pxc-strict-mode
+
+# let's cache the default value first.
+#--let $pxc_strict_mode_saved = `SELECT @@pxc_strict_mode`
+
+--echo # set default value
+--error ER_WRONG_VALUE_FOR_VAR
```

Comment:
Right. This turned out bit funny.
We don't allow switching pxc-strict-mode if server is running as standalone.
(sys_vars run the server in non-wsrep mode).

Then why do we have this TCs and how do we check toggling of pxc-strict-mode.
(toggling is being check as part of pxc-strict-mode.test and we have 2 options with this tcs.
a. remove it  but that means adding exception as every variable should have test-case as per the existing enforced mysql semantics.
b. this tcs still help us make sure that value that is being passes is invalid and not garbage.
Garbage will produce SEGFAULT)